src/main/java/id/ac/itb/logistik/ditlog/model/User.java

```diff
@@ -10,12 +10,6 @@
 @Entity
 @Table(name = "SILOG_USER")
 public class User {
```

Comment:
try merging master into this branch first before proceeding (resolve merge conflict first)